src/ripple/app/misc/AmendmentTable.h

```diff
@@ -53,19 +95,25 @@ class AmendmentSet
 class AmendmentState
 {
 public:
-    bool mVetoed;   // We don't want this amendment enabled
+    bool mVetoed;  // We don't want this amendment enabled
     bool mEnabled;
     bool mSupported;
     bool mDefault;  // Include in genesis ledger
 
-    core::Clock::time_point m_firstMajority; // First time we saw a majority (close time)
-    core::Clock::time_point m_lastMajority;  // Most recent time we saw a majority (close time)
+    core::Clock::time_point
+        m_firstMajority;  // First time we saw a majority (close time)
+    core::Clock::time_point
+        m_lastMajority;  // Most recent time we saw a majority (close time)
 
     std::string mFriendlyName;
 
     AmendmentState ()
-        : mVetoed (false), mEnabled (false), mSupported (false), mDefault (false),
-          m_firstMajority (0), m_lastMajority (0)
+        : mVetoed (false)
```

Comment:
If all these variables were "inline initialized" then the constructor is reduced to: `AmendmentState () = default`